entity-service/src/main/resources/configs/common/application.conf

```diff
@@ -22,6 +22,7 @@ attribute.service.config = {
 }
 
 entity.service.change.skip.attributes = []
+eqs.change.notification.enabled.entity.types = ["*"]
```

Comment:
We should have control for both EQS and EDS. May be even a single config control for both. Otherwise, it becomes inconsistent

We should also club entity change related configs into a single section. May be something like

```
entity.service.change {
  allowedEntityTypes = [`*`]
  skipAttributes = []
}
```

Reply:
I knew it this was coming 😂 . So the thing is I am not sure how much impact all this change will have on the scale. Feels like we need at least in the start more control over the behavior of EQS, otherwise, we might run into the same issue again. I checked the traffic over dogfooding and lots seem like time updates only. Let me change it. will monitor it more carefully when deployed